src/wagtail_personalisation/wagtail_hooks.py

```diff
@@ -139,9 +140,13 @@ def _check_for_variations(segments, page):
     :rtype: wagtail_personalisation.models.PersonalisablePage or None
 
     """
+    try:
+        model = AbstractPersonalisablePage.__subclasses__()[0]
```

Comment:
Won't this mean that a project can only ever have one valid page type inheriting from `AbstractPersonalisablePage`? That seems like a very severe restriction.